modules/ROOT/pages/6.8-release-notes.adoc

```diff
@@ -291,6 +291,45 @@ waiting for confirmation on these new feature tickets.
 === Search and replace plugin would incorrectly find matching text inside SVG elements.
 // #TINY-10162
 
+=== Merging an external p inside a list via delete or backspace would incorrectly try to move a parent element inside a child element.
+// #TINY-10289
+Previously, when pressing backspace or delete on a paragraph, and while its also nested inside a `<li>` element, would move the paragraph element inside the child element, rather than merging the context into the `<li>` child element context.
```

Comment:
only one thing about this, considering the example below here we have 2 situations:

1) you are at the beginning of the `p` and you press `backspace`
2) you are at the end of the `li` with id `child` and you press `delete`

I don't know if there is a better way to make it clear the distinction

Reply:
```suggestion
Previously, when the caret was positioned at either the,

. **beginning** of the `<p>` and the `backspace` key is pressed; or
. **end** of the `<li>` with **id child** and the `delete` key is pressed.

the `<p>` tag element would be moved inside the child element, rather than the context being merged with the `<li>` child element context.
```